src/js/flow/acquia_lift.variations.views.js

```diff
@@ -45,7 +45,7 @@
        *   True if showing the highlight, false if no highlight should be shown.
        */
       highlightAnchor: function(show) {
-        var highlightClass = 'acquia-lift-page-variation-item';
```

Comment:
You'll find this in your next round of work but this class is referred to by the qunit page_variations.tests.js.